### PR TITLE
[v1.x] Cherry-pick #17776 #17681

### LINKED ISSUE #18462: [v1.7] PyPI description of nightly wheels outdated
The PyPI description for v1.7 nightly wheels is outdated. The wheels use the non MKL-DNN versions of doc files for each flavor.

This is where doc files are referenced:
https://github.com/apache/incubator-mxnet/blob/v1.7.x/tools/pip/setup.py#L116

The doc folder needs to be updated as per #17776 #17681 
https://github.com/apache/incubator-mxnet/tree/v1.7.x/tools/pip/doc

### PR DESCRIPTION
**Pull request by mseth10**
## Description ##
Cherry-pick #17776 #17681 to update PyPI description

This fixes #18462

### CODE REVIEW

**Comment by mseth10** [comment]
@leezu please help review and merge this PR. This fixes #18462

File: cd/mxnet_lib/dynamic/Jenkins_pipeline.groovy [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 // NOTE: the following variables are referenced in the mxnet_lib_pipeline jenkins file imported bellow
 // libmxnet location
-libmxnet = 'lib/libmxnet.so'
```
Comment (leezu) [inline cd/mxnet_lib/dynamic/Jenkins_pipeline.groovy:27]: As with the 1.7 PR: Changing this line is done when switching to cmake build on CD. I don't think it's done yet (but we certainly want to do it at some point to decrease the infrastructure code divergence between 1.x and master branch)

Comment (mseth10) [reply cd/mxnet_lib/dynamic/Jenkins_pipeline.groovy]: I reverted this line. Rest of the code changes look good?

**Comment by leezu** [comment]
Clarification of the legal status of the pipeline as of early 2019 is at https://issues.apache.org/jira/browse/LEGAL-427 The major change since then is that the pipeline is managed as part of the https://github.com/apache/incubator-mxnet/ and executed by the Jenkins Server. This may put it under control of the PPMC. If that is true and if that affects it's status as third-party releases pends clarification of ASF Legal in https://issues.apache.org/jira/browse/LEGAL-515

**Comment by ciyongch** [comment]
Hi @mseth10  @leezu , seems v1.7.x branch also need this PR to update the pypi desc, can you help to backport to v1.7.x as well. Thanks.

**Comment by mseth10** [comment]
@ciyongch Created another PR for 1.7 branch #18464

**Comment by mseth10** [comment]
@mxnet-bot run ci [unix-gpu]

**Comment by mseth10** [comment]
@mxnet-bot run ci [unix-gpu]

**Comment by mseth10** [comment]
@mxnet-bot run ci [unix-gpu]

**Comment by mseth10** [comment]
@mxnet-bot run ci [unix-gpu]

**Comment by mseth10** [comment]
@mxnet-bot run ci [unix-gpu]

**Comment by mseth10** [comment]
@mxnet-bot run ci [unix-gpu]

**Comment by mseth10** [comment]
@leezu please merge.

**Comment by ciyongch** [comment]
Ping @TaoLv , @pengzhao-intel to help merge :)

**Pull request merged**